nix/ovpn.nix

```diff
@@ -0,0 +1,43 @@
+{ config, pkgs, ... }:
+{
+  boot = {
+    kernel.sysctl = {
+      "net.ipv4.ip_forward" = 1;
+    };
+  };
+  networking = {
+    firewall = {
+      enable = true;
+      extraCommands = ''
+        iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eno1 -j MASQUERADE
+      '';
+      extraStopCommands = ''
+        iptables -t nat -D POSTROUTING -s 10.8.0.0/24 -o eno1 -j MASQUERADE
+      '';
+      allowedUDPPorts = [ 1195 ]; # openvpn
+      trustedInterfaces = [ "tun1" ];
+    };
+  };
+  services = {
+    openvpn = {
+      servers = {
+        for-phone = {
```

Comment:
Is this actually "for-phone"? Do you make one server per device?

Thats a bit unpleasant.